### PR TITLE
added proto for loopdev and ramdisk

### PR DESCRIPTION
**Pull request by antonipx**
Important: I only changed `api/api.proto` and `cli/cli_test.go`. I see changes in `flexvolume.pb.gw.go` and `api.pb.go` which I did not make. I'm unsure of their impact at present.

### CODE REVIEW

File: cli/cli_test.go [hunk 1]
```diff
@@ -34,7 +34,9 @@ func TestCmdMarshalProto(t *testing.T) {
  "sticky": false,
  "group_enforced": false,
  "compressed": false,
- "cascaded": false
+ "cascaded": false,
+ "loopdev": false,
+ "ramdisk": false
```
Comment (lpabon) [inline cli/cli_test.go]: Are you testing that the values are always false? I think you should test for setting values to true or false, right? Who is going to take action with these values?

**Comment by lpabon** [comment]
@antonipx Could you describe why these are needed?

**Comment by antonipx** [comment]
@lpabon The changes are needed to implement two features for Portworx. Loopback device on top of PXD volume and ramdisk as backing store. Proto changes are required for these to be instrumented via pxctl, etc.

**Comment by lpabon** [comment]
@antonipx Thank for the description. What tests need to be written to test this?

**Comment by ggriffiths** [comment]
Closing manually as we're closing old PRs before Jan 2020. Feel free to update and re-open if this is still relevant.

**Pull request closed without merging**